### PR TITLE
Update description of toolchain and new compilers.

### PR DESCRIPTION
**Pull request by pmlandwehr**
Fixes for clarity. (Hopefully.)

### CODE REVIEW

**Comment by jakirkham** [comment]
So these changes should be made to [`src/meta.rst`]( https://github.com/conda-forge/conda-forge.github.io/blob/master/src/meta.rst ). The file being edited here is pregenerated from that file.

File: docs/_sources/meta.rst.txt [hunk 1]
```diff
@@ -167,9 +167,9 @@ You need to rerender the feedstock after this change.
...
+While ``conda-build=3`` provides access to Anaconda 5 compilers, ``conda-forge`` is not using them yet. This is because packages built using the Anaconda 5 compilers are sometimes incompatible with those built with the older compilers in the CI platforms conda-forge has been using. As a general rule, all of a package's dependencies should be compiled with the new compiler before using the new compiler to recompile the package itself. This target hasn't yet been met for all ``conda-forge`` packages, but a large-scale rebuild-and-reupload using the new compilers is planned for an unspecified future date.
 
-However, using the ``{{ compiler('cxx') }}`` is supported in ``conda-forge``, but it installs the ``toolchain`` package which activates the compilers in the CI environment. If you were using ``toolchain`` or ``gcc`` build deps, consider using the following,
+Using the ``{{ compiler('cxx') }}`` syntax is supported in ``conda-forge``, but it installs the ``toolchain`` package which activates the compilers in the CI environment. If you have been using ``toolchain`` or ``gcc`` build dependencies, consider using the following instead:
```
Comment (dougalsutherland) [inline docs/_sources/meta.rst.txt]: I'd rephrase to be more encouraging of using the new syntax. We want people using that to make the future transition easier....

**Comment by scopatz** [comment]
pinging this @pmlandwehr

**Comment by ocefpaf** [comment]
B/c this is a temporary thing and after the migration all recipes will look like this do we need to write that in the docs?

**Comment by henryiii** [comment]
This part of the documentation is currently unclear, so anything would help. I have a package that requires GCC 4.8.5 or better, and fails to build on GCC 4.8.2. As long as the dependencies are met, it should be possible to build with the new compilers, but it is not clear from the documentation or the example repository.

**Comment by croth1** [comment]
I will close this as most information is not up-to-date anymore
